src/Microsoft.Data.Analysis/ColumnArithmeticTemplate.ttinclude

```diff
@@ -38,6 +38,56 @@
         return $"{keyword} (typeof(T) == typeof({type.TypeName}))";
     }
 
+        // A way to discern implicit conversions. For ex: short has primitivitty 2. int has primitivitty 3. primitivitty(short) + primitivitty(int) > 2 * primitivitty(short) implying that a conversion has to take place 
```

Comment:
Are these changes necessary/intended?

Reply:
Yup. `Converters.tt` uses this map to decide which APIs to generate.